runelite-client/src/main/java/net/runelite/client/ui/overlay/components/PanelComponent.java

```diff
@@ -111,6 +117,8 @@ public Dimension render(Graphics2D graphics)
 			preferredSize.width - border.x - border.width,
 			preferredSize.height - border.y - border.height);
 
+		final Point boundsLoc = new Point(parentBoundsLocation.x + preferredLocation.x, parentBoundsLocation.y + preferredLocation.y);
```

Comment:
I dont like this parentBounds thing. You can just do that by yourself outside of panel. Or if you really want to implement this properly, add getBounds() to LayoutableRenderableEntity and implement it in other implementations too.

Reply:
See https://github.com/runelite/runelite/pull/7705